tests/failing-run-make-tests.txt

```diff
@@ -30,4 +30,13 @@ tests/run-make/rustdoc-themes
 tests/run-make/rustdoc-verify-output-files
 tests/run-make/rustdoc-with-out-dir-option
 tests/run-make/rustdoc-with-output-option
-tests/run-make/rustdoc-with-short-out-dir-option
\ No newline at end of file
+tests/run-make/rustdoc-with-short-out-dir-option
+tests/run-make/return-non-c-like-enum
+tests/run-make/lto-smoke-c
+tests/run-make/issue-68794-textrel-on-minimal-lib
+tests/run-make/issue-36710
+tests/run-make/glibc-staticlib-args
+tests/run-make/foreign-exceptions
+tests/run-make/foreign-double-unwind
+tests/run-make/c-link-to-rust-staticlib
+tests/run-make/arguments-non-c-like-enum
```

Comment:
It seems you might only need to install the package to be able to link `lstdc++` to have these tests pass in the CI.

Reply:
@antoyo sorry, I don't understand, what package are you referring to? 
And would this resolve only these 9 or all failing tests?

Reply:
The package is named either `libstdc++-12-dev` or `libstdc++-11-dev`.
I only saw the failures for those 9 tests, so probably only these one (perhaps not all of these 9 tests).